notebooks/data_validation.ipynb

```diff
@@ -0,0 +1,1243 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/gbotemi/miniconda3/envs/pypsa-earth/lib/python3.10/site-packages/pypsa/networkclustering.py:16: UserWarning:\n",
+      "\n",
+      "The namespace `pypsa.networkclustering` is deprecated and will be removed in PyPSA v0.24. Please use `pypsa.clustering.spatial instead`. \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import pypsa\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pycountry\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Choose the country and the year for validation\n",
+    "# Country code should be in ISO 3166-1 alpha-2 format which is 2 country code letters.\n",
+    "# For example, Germany is DE, France is FR, etc.\n",
+    "country_code = \"US\"\n",
+    "scenario_folder = \"US\"\n",
```

Comment:
Hi, @GbotemiB. Really a great job. Tested locally everything works fine. One thing I would suggest here to change scenario_folder to `US_2021`, because in calibration data provided we have folder names as `US_2021`. If you fix that, I can merge it.

Reply:
Also please pull the latest changes.

Reply:
@GbotemiB, another thing regarding plot location. Can you save plots not to `results`, but `results/plots`?

Reply:
Hi @yerbol-akhmetov, I have made the changes to the notebook. 
Thank you for the review.

Reply:
@GbotemiB, great, thanks!